### ISSUE TITLE
POST - ParticipantsByTypeAndID - http code and status code wrong

### ISSUE THREAD

**Issue by amarmodus** [mojaloop/project#1246]
**Summary**:
POST - Participants By Type and ID 
1. With Missing mandatory date header - 400 expected with 3102 error code
<img width="1040" alt="Screenshot 2020-02-27 01 22 10" src="https://user-images.githubusercontent.com/53434415/75403449-aff20500-58ff-11ea-9a28-241ad7a3d5d7.png">

2. With Malformed Type - 400 expected with 3101 error code
<img width="1035" alt="Screenshot 2020-02-27 01 22 25" src="https://user-images.githubusercontent.com/53434415/75403454-b3858c00-58ff-11ea-852c-8920f1768fd7.png">

3. With Missing mandatory fspiop header - 400 expected with 3102 error code
<img width="1049" alt="Screenshot 2020-02-27 01 23 55" src="https://user-images.githubusercontent.com/53434415/75403507-d7e16880-58ff-11ea-9ad7-385ca34813c6.png">

4.With Missing content type - 400 expected with 3102 error code
<img width="1037" alt="Screenshot 2020-02-27 01 25 30" src="https://user-images.githubusercontent.com/53434415/75403589-0fe8ab80-5900-11ea-8d1a-9e5063ef7c5c.png">

5.With Missing content Type header - 400 expected with 3102 error code
<img width="1054" alt="Screenshot 2020-02-27 01 32 50" src="https://user-images.githubusercontent.com/53434415/75404020-33f8bc80-5901-11ea-980e-e5cdddc372e4.png">

**Severity**:
Medium

**Priority**:
Medium

**Expected Behavior**

**Steps to Reproduce**
```
curl --location --request POST 'http://dev1-account-lookup.mojaloop.live/participants/MSISDN/27713803912' \
--header 'Date: Thu, 27 Feb 2020 01:23:34 GMT' \
--header 'FSPIOP-Source: payerfsp' \
--header 'Content-Type: application/vnd.interoperability.participants+json;version=1.0' \
--data-raw '{
	"fspId": "payeefsp",
	"currency": "USD"
}'
```

**Specifications**
  - Component (if known):
  - Version:8.8.0
  - Platform:
  - Subsystem:ALS
  - Type of testing:Postman
  - Bug found/raised by:@amarmodus

**Notes**:

- Severity when opened:
- Priority when opened:

**Comment by elnyry** [comment]
@amarmodus Fixing #1245 and #1243 may have fixed this as well. Please confirm

**Comment by amarmodus** [comment]
<img width="1042" alt="Screenshot 2020-02-28 13 35 59" src="https://user-images.githubusercontent.com/53434415/75552998-6ef90e00-5a2f-11ea-92eb-ef01959658e2.png">
Validated in Dev 2
This is available with ALS v9.2.3: https://github.com/mojaloop/account-lookup-service/releases/tag/v9.2.3

**Comment by elnyry** [comment]
Thanks @amarmodus
